pages/checkout/index.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <h1 class="mb-11">Checkout</h1>
+  <div class="grid grid-cols-2 gap-x-72 mb-80">
+    <CheckoutDetails />
+    <CheckoutSummary />
+  </div>
+</template>
+
+<script setup lang="ts"></script>
```

Comment:
I'm getting this warning on this page:

![image](https://user-images.githubusercontent.com/18750939/222709442-8c36b5bb-005f-4831-bc4c-f6c741a43589.png)